release/release.go

```diff
@@ -168,6 +171,36 @@ func getWSLVersion() int {
 	return 2
 }
 
+type snapYaml struct {
+	Name string `yaml:"name"`
+}
+
+// Get the name of the base snap if running inside one.
```

Comment:
this does not follow go conventions for docs, it should start with the name of the functions, given that is not exported I we are not super picky about it but then we tend to make our one-off comments all lowercase without a dot at the end.